frappe/printing/doctype/print_format/print_format.js

```diff
@@ -40,20 +40,23 @@ frappe.ui.form.on("Print Format", {
 				frm.set_df_property("html", "reqd", 1);
 			}
 			if (frappe.model.can_write("Customize Form")) {
-				frappe.db.get_value("DocType", frm.doc.doc_type, "default_print_format", (r) => {
-					if (r.default_print_format != frm.doc.name) {
-						frm.add_custom_button(__("Set as Default"), function () {
-							frappe.call({
-								method: "frappe.printing.doctype.print_format.print_format.make_default",
-								args: {
-									name: frm.doc.name,
-								},
-								callback: function () {
-									frm.refresh();
-								},
-							});
-						});
+				frappe.model.with_doctype(frm.doc.doc_type, function () {
+					let current_format = frappe.get_meta(frm.doc.DocType).default_print_format;
```

Comment:
@ankush, shouldn't this be `let current_format = frappe.get_meta(frm.doc.doc_type).default_print_format;`?

Reply:
Corrected, thanks for pointing out. 

https://github.com/frappe/frappe/commit/c69c040186e76c5798ac9c2d5d15b545a61eda27